### PR TITLE
Update `wgpu` example to `wgpu` 0.19

### PR DESCRIPTION
**Pull request by ids1024**

### CODE REVIEW

**Comment by ids1024** [comment]
Hm, so wgpu requires Rustc 1.70?

It's not really a problem for the example to require a higher MSRV than the library, but it's a bit awkward to CI test that.

**Comment by i509VCB** [comment]
> Hm, so wgpu requires Rustc 1.70?
> 
> It's not really a problem for the example to require a higher MSRV than the library, but it's a bit awkward to CI test that.

What wgpu uses is largely based on Firefox.

I'd be fine bumping to 1.70. Although didn't rwh bump up and I know ash now needs 1.69 on git master.

File: examples/wgpu.rs [hunk 1]
```diff
@@ -231,6 +215,7 @@ impl WindowHandler for Wgpu {
             alpha_mode: wgpu::CompositeAlphaMode::Auto,
             width: self.width,
             height: self.height,
+            desired_maximum_frame_latency: 2,
```
Comment (i509VCB) [inline examples/wgpu.rs:218]: Where is this field being used?

**Review by i509VCB — APPROVED** [review]

File: examples/wgpu.rs (hunk 1 above)
Comment (ids1024) [reply examples/wgpu.rs:218]: This is a field of `wgpu::SurfaceConfiguration`. That used to be hard-coded to 2, but is now configurable.

Comment (i509VCB) [reply examples/wgpu.rs:218]: Oh okay so the web UI probably didn't show the struct owning the field.

**Pull request merged**